From 57700b94f9111578d0fc05bb8f273c0b29951572 Mon Sep 17 00:00:00 2001
From: Rajesh Bhagat <rajesh.bhagat@nxp.com>
Date: Wed, 11 May 2016 14:59:39 +0530
Subject: [PATCH 29/93] armv8: ls1012a: Added CSU assignment for USB2

Access settings for USB2 IP is added through CSU register.

Added CSU ID for USB2, reg: CSL23_REG[8:0]

Signed-off-by: Rajesh Bhagat <rajesh.bhagat@nxp.com>
---
 .../include/asm/arch-fsl-layerscape/ns_access.h    |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/include/asm/arch-fsl-layerscape/ns_access.h b/arch/arm/include/asm/arch-fsl-layerscape/ns_access.h
index d6642a7..2fd33e1 100644
--- a/arch/arm/include/asm/arch-fsl-layerscape/ns_access.h
+++ b/arch/arm/include/asm/arch-fsl-layerscape/ns_access.h
@@ -38,6 +38,7 @@ enum csu_cslx_ind {
 	CSU_CSLX_ESDHC,
 	CSU_CSLX_IFC = 45,
 	CSU_CSLX_I2C1,
+	CSU_CSLX_USB_2,
 	CSU_CSLX_I2C3 = 48,
 	CSU_CSLX_I2C2,
 	CSU_CSLX_DUART2 = 50,
@@ -117,6 +118,7 @@ static struct csu_ns_dev ns_dev[] = {
 	 {CSU_CSLX_ESDHC, CSU_ALL_RW},
 	 {CSU_CSLX_IFC, CSU_ALL_RW},
 	 {CSU_CSLX_I2C1, CSU_ALL_RW},
+	 {CSU_CSLX_USB_2, CSU_ALL_RW},
 	 {CSU_CSLX_I2C3, CSU_ALL_RW},
 	 {CSU_CSLX_I2C2, CSU_ALL_RW},
 	 {CSU_CSLX_DUART2, CSU_ALL_RW},
-- 
1.7.9.5

